LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Chandler Weiner
+Copyright (c) 2019 Chandler Weiner
```

Comment:
Could you please change this to 2018-2019?

Reply:
Maybe like this?

```suggestion
Copyright (c) 2018-present Chandler Weiner
```

Reply:
Hummm....I'm not sure. I'm not a lawyer, but there's something in the back of my brain that says that copyright has to expire at some point, but I genuinely don't know. :man_shrugging: Not like there's any true copyright claim to a text list anyway...